sphinx_search/static/js/rtd_sphinx_search.js

```diff
@@ -124,6 +118,18 @@ const updateSearchBar = () => {
 };
 
 
+/*
+ * Returns true if the modal window is visible.
+ */
+const isModalVisible = () => {
+  let modal = document.querySelector(".search__outer__wrapper");
+  if (modal !== null && modal.style !== null && modal.style.display !== null) {
```

Comment:
Soon you'll be able to use `modal?.style?.display` syntax here instead of this chain of conditionals, but it's a relatively new addition